### PR TITLE
Add Incopro interview

### PR DESCRIPTION
**Pull request by brian-brazil**

### CODE REVIEW

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+## What do you think the future holds for Incorpo and Prometheus?
+
+Making Prometheus scale or at least having multiple instances is helping us
+being more robust. The Alertmanager’s HA solution is definitely a welcome
+addition. The toughest job for us is to realize which metrics are the most
+important for us. Developers will need to output their metrics in the
+Prometheus way, which has not started, but the Ops team is pushing for these
+changes. We are going to be transitioning to Docker based dynamically
+distributed instances, monitoring these will going to be essential and
+prometheus is going to be proper tool for it, as mentioned even at PromCon, it
```
Comment (grobie) [inline content/blog/2016-09-28-interview-with-incopro.md]: Prometheus

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+was designed for the dynamic world. As business grows so will the teams and
+educating people about the use of prometheus is going to be a high priority
```
Comment (grobie) [inline content/blog/2016-09-28-interview-with-incopro.md]: Prometheus

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+title: Interview with Incopro
+created_at: 2016-09-28
+kind: article
+author_name: Brian Brazil
+---
+
+*Continuing our series of interviews with users of Prometheus, INCOPRO talks
+about how they went from Zabbix and Monit to Prometheus.*
+
+## Can you tell us about yourself and what Incorpo does?
```
Comment (beorn7) [inline content/blog/2016-09-28-interview-with-incopro.md]: Incopro. (and why not all-caps here but everywhere else?)

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+[INCOPRO](http://www.incopro.co.uk/) is a brand protection company. It uses
+technology, combined with IP and investigation expertise, to deliver effective
+and targeted protection to businesses looking to protect their IP assets
+online. Its Talisman brand protection system is used by many of the largest
+global brands to track and assess infringement issues across Marketplaces,
+Websites, Social Media, App Stores and more. INCOPRO’s expert analyst team use
```
Comment (beorn7) [inline content/blog/2016-09-28-interview-with-incopro.md]: Why are those capitalized? (Including Marketplaces on the previous line.)

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+At first we tried to mimic the old monitoring system, which was a good learning
+curve, get the same kind of alerts we were getting from Zabbix, this involved
+using the [blackbox_exporter](https://github.com/prometheus/blackbox_exporter)
+and some metrics from
+[node_exporter](https://github.com/prometheus/node_exporter). At that time, as
+far as I remember, Pushover was not available, thus the alerting was not
+possible and we didn’t want to transition to anything else. Other useful
+metrics showed up on our dashboards which included our scrapers. We could
+finally visualize how much data are we collecting, troubleshooting improved a
```
Comment (beorn7) [inline content/blog/2016-09-28-interview-with-incopro.md]: "are we" → "we are", no?

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+lot, we could speed up our whole collection time based on these valuable
+metrics. Although during this time we were using the
+[Pushgateway](https://github.com/prometheus/pushgateway/) way too much, we
+managed to make the data valuable for us.
+
+![The state of the system](/assets/blog/2016-09-28/devops-dashboard.PNG)
+
+## What improvements have you seen since switching?
+
+After the whole alerting was switched over to Prometheus+Alertmanager and we
```
Comment (beorn7) [inline content/blog/2016-09-28-interview-with-incopro.md]: "+" → " and "

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+tossed Zabbix out, we never looked back. Beside the previously mentioned
+scraping problems, Prometheus helped us realise which servers were over or
+underused, having ~150 barebone servers across the world makes it hard to
+realize where are you over or underperforming. The Grafana dashboards were
```
Comment (beorn7) [inline content/blog/2016-09-28-interview-with-incopro.md]: Same as before: "are you" → "you are"

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+easier to get by as we were sure we would be alerted if something went wrong.
+We were finally able to monitor our whole infrastructure, Cassandra,
+Elasticsearch, MySQL, RabbitMQ, collection and analysis servers, proxies,
+frontend and other backend servers. The best way to tell how big of an impact
+Prometheus had on our business is to mention how fast some of our team members
+notice if new data is not showing up on our dashboards.
+
+![Metrics from one of the 4 cassandra datacenters](/assets/blog/2016-09-28/cassandra-dashboard.PNG)
+
+## What do you think the future holds for Incorpo and Prometheus?
```
Comment (beorn7) [inline content/blog/2016-09-28-interview-with-incopro.md]: Incopro (and all-caps question as before).

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+Prometheus way, which has not started, but the Ops team is pushing for these
+changes. We are going to be transitioning to Docker based dynamically
+distributed instances, monitoring these will going to be essential and
+prometheus is going to be proper tool for it, as mentioned even at PromCon, it
+was designed for the dynamic world. As business grows so will the teams and
+educating people about the use of prometheus is going to be a high priority
+soon, even our brand enforcement analysts could use the metrics provided
+through Prometheus to showcase their efficiency. We had discussions on
+outputting data from Prometheus to our clients so they know which markets are
+we watching for them and how efficient are we.
```
Comment (beorn7) [inline content/blog/2016-09-28-interview-with-incopro.md]: Twice "are we" → "we are"

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+day to day operations in the short term. I became the only Ops at the company
+so everything falls under my jurisdiction.
+
+
+## What was your pre-Prometheus monitoring experience?
+
+There wasn’t a real solution, especially no alerting in place.
+[Zabbix](http://www.zabbix.com/) and [Monit](https://mmonit.com/) have been
+used to some extent, when things started to get out of control as the company
+grew bigger a new solution was needed and Prometheus was found back in December
```
Comment (juliusv) [inline content/blog/2016-09-28-interview-with-incopro.md]: There are some commas or other things missing here.

How about something like: "control. As the company grew bigger, a new solution was needed..."

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+2015.
+
+## How did you transition?
+
+At first we tried to mimic the old monitoring system, which was a good learning
+curve, get the same kind of alerts we were getting from Zabbix, this involved
```
Comment (juliusv) [inline content/blog/2016-09-28-interview-with-incopro.md]: is there a word or something missing before "get"? In any case, it's a too long run-on sentence.

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+using the [blackbox_exporter](https://github.com/prometheus/blackbox_exporter)
+and some metrics from
+[node_exporter](https://github.com/prometheus/node_exporter). At that time, as
+far as I remember, Pushover was not available, thus the alerting was not
+possible and we didn’t want to transition to anything else. Other useful
+metrics showed up on our dashboards which included our scrapers. We could
```
Comment (juliusv) [inline content/blog/2016-09-28-interview-with-incopro.md]: Comma before "which"

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+finally visualize how much data are we collecting, troubleshooting improved a
+lot, we could speed up our whole collection time based on these valuable
```
Comment (juliusv) [inline content/blog/2016-09-28-interview-with-incopro.md]: we -> and we

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+[Pushgateway](https://github.com/prometheus/pushgateway/) way too much, we
+managed to make the data valuable for us.
+
+![The state of the system](/assets/blog/2016-09-28/devops-dashboard.PNG)
+
+## What improvements have you seen since switching?
+
+After the whole alerting was switched over to Prometheus+Alertmanager and we
+tossed Zabbix out, we never looked back. Beside the previously mentioned
+scraping problems, Prometheus helped us realise which servers were over or
```
Comment (juliusv) [inline content/blog/2016-09-28-interview-with-incopro.md]: "over" -> "over-"

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 15]
```diff
@@ -0,0 +1,94 @@
...
+underused, having ~150 barebone servers across the world makes it hard to
```
Comment (juliusv) [inline content/blog/2016-09-28-interview-with-incopro.md]: Another too long run-on sentence... Period after "underused"?

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 16]
```diff
@@ -0,0 +1,94 @@
...
+![Metrics from one of the 4 cassandra datacenters](/assets/blog/2016-09-28/cassandra-dashboard.PNG)
+
+## What do you think the future holds for Incorpo and Prometheus?
+
+Making Prometheus scale or at least having multiple instances is helping us
+being more robust. The Alertmanager’s HA solution is definitely a welcome
+addition. The toughest job for us is to realize which metrics are the most
+important for us. Developers will need to output their metrics in the
+Prometheus way, which has not started, but the Ops team is pushing for these
+changes. We are going to be transitioning to Docker based dynamically
```
Comment (juliusv) [inline content/blog/2016-09-28-interview-with-incopro.md]: Docker based -> Docker-based

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 17]
```diff
@@ -0,0 +1,94 @@
...
+distributed instances, monitoring these will going to be essential and
+prometheus is going to be proper tool for it, as mentioned even at PromCon, it
+was designed for the dynamic world. As business grows so will the teams and
```
Comment (juliusv) [inline content/blog/2016-09-28-interview-with-incopro.md]: Put some periods in this long run-one sentence...

Comment (juliusv) [inline content/blog/2016-09-28-interview-with-incopro.md]: teams -> teams,

File: content/blog/2016-09-28-interview-with-incopro.md [hunk 18]
```diff
@@ -0,0 +1,94 @@
...
+educating people about the use of prometheus is going to be a high priority
+soon, even our brand enforcement analysts could use the metrics provided
```
Comment (juliusv) [inline content/blog/2016-09-28-interview-with-incopro.md]: , even -> . Even

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 9 above)
Comment (juliusv) [inline content/blog/2016-09-28-interview-with-incopro.md]: (yes, twice in this sentence)

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 3 above)
Comment (brian-brazil) [inline content/blog/2016-09-28-interview-with-incopro.md]: Fixed. Looking at their website, their branding is the usual capitalization so changed.

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 1 above)
Comment (brian-brazil) [inline content/blog/2016-09-28-interview-with-incopro.md]: Done

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 6 above)
Comment (brian-brazil) [inline content/blog/2016-09-28-interview-with-incopro.md]: That wouldn't read well with the and right after it

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 4 above)
Comment (brian-brazil) [inline content/blog/2016-09-28-interview-with-incopro.md]: Changed

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 5 above)
Comment (brian-brazil) [inline content/blog/2016-09-28-interview-with-incopro.md]: That reads okay to me

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 9 above)
Comment (brian-brazil) [inline content/blog/2016-09-28-interview-with-incopro.md]: This reads fine for me.

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 10 above)
Comment (brian-brazil) [inline content/blog/2016-09-28-interview-with-incopro.md]: Done

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 11 above)
Comment (brian-brazil) [inline content/blog/2016-09-28-interview-with-incopro.md]: Fixed.

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 12 above)
Comment (brian-brazil) [inline content/blog/2016-09-28-interview-with-incopro.md]: Fixed

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 13 above)
Comment (brian-brazil) [inline content/blog/2016-09-28-interview-with-incopro.md]: Fixed

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 14 above)
Comment (brian-brazil) [inline content/blog/2016-09-28-interview-with-incopro.md]: Fixed

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 15 above)
Comment (brian-brazil) [inline content/blog/2016-09-28-interview-with-incopro.md]: Seems okay to me.

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 16 above)
Comment (brian-brazil) [inline content/blog/2016-09-28-interview-with-incopro.md]: Changed

File: content/blog/2016-09-28-interview-with-incopro.md (hunk 17 above)
Comment (brian-brazil) [inline content/blog/2016-09-28-interview-with-incopro.md]: Tweaked

**Comment by beorn7** [comment]
About the many word order issues: I have to emulate Gabe and correct the native speaker – subordinate questions have no inversion.
https://jakubmarian.com/do-you-know-where-it-isis-it-word-order-in-subordinate-clauses-in-english/
http://www.grammar-once-and-for-all.com/syntax/wh-word-clauses/
http://english.stackexchange.com/questions/254521/word-order-for-subordinate-questions

**Comment by juliusv** [comment]
Yeah, :+1: for correcting those.

**Pull request merged**